metadata/metadata_tutorial_association.Rmd

```diff
@@ -0,0 +1,541 @@
+---
+title: 'Metadata Upload Tutorial Part 2: Associations'
+author: "Kimberly Huynh"
+date: "4/30/2019"
+output: html_document
+---
+
+# Section 1: Overview
+
+The objective of this tutorial is to demonstrate how to associate metadata that you have added to the traits database. 
```

Comment:
clarify ... the term "Associations" in the title and "associate metadata" here may not be clear to the end user ... maybe